src/Maker/MakeResetPassword.php

```diff
@@ -111,6 +112,7 @@ public function configureDependencies(DependencyBuilder $dependencies): void
         $dependencies->addClassDependency(Validation::class, 'symfony/validator');
         $dependencies->addClassDependency(SecurityBundle::class, 'security-bundle');
         $dependencies->addClassDependency(AppVariable::class, 'twig');
+        $dependencies->addClassDependency(TranslatorInterface::class, 'symfony/translation', false);
```

Comment:
Do we want to add this here? I believe the end result is if you are missing any required dependencies, then the `composer require` will now include `symfony/translation` in the error message. But, the user doesn't NEED to install this package, and I think the intention is just to detect IF they have it, so that we can know how to generate the controller.

Reply:
@weaverryan The intention is meant to be exactly like you described! If users have a translator installed - they will have translated exceptions. But without this line tests were failing somehow... and I passed `false` as the 3rd argument there that already says this dependency is "optional" and not "required". Is it enough? Or do I understand that 3rd option wrong and it's not about specifying required/optional dependency?

Reply:
Ah, if it was to get the tests working, then try adding `->addExtraDependencies()` in the test case - e.g. https://github.com/symfony/maker-bundle/blob/112ed509fc7753a3d4f555f4e491a04213ee79ac/tests/Maker/MakeEntityTest.php#L81

Normally, when you run a test, it will find all of the dependencies configured in this method and will install those. So probably this solution worked because it caused the `symfony/translator` to be installed. BUT, we don't really want to recommend it as a package, so probably the better thing to do is to *not* include it here, but then hint to the test system that it SHOULD install it.

However, if you didn't include it here AND you didn't include it in `setExtraDependencies()`, in theory, the tests should NOT have failed. In that situation, the test would have been testing the "what happens of `symfony/translator` is not installed situation". We may need to have one test that does NOT include `symfony/translator` and another that DOES include it, and we make sure both work.

Reply:
OK, failed tests were valid due to empty line, fixed in a00d568

And thanks for the tip about `addExtraDependencies()`! I added a new test to cover the case installed Translator in 6e64745